recipe/run_conda_forge_build_setup_win

```diff
@@ -8,5 +8,5 @@ conda install -n root --yes --quiet jinja2 anaconda-client
 
 # see https://github.com/conda-forge/staged-recipes/pull/750#issuecomment-225165530
 conda install -n root --yes --quiet conda-build=1.20.1
-
+if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
```

Comment:
What comment? Above there is one mentioning that we need to fix `conda-build-all`, and all the issues with 1.20.1 are already fixed in the latest `conda-build`, no? (Actually we never tested it as the pinning here remained...)